templates/Exception/error.html.twig

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html.twig' %}
+{% extends '../../../../../templates/base.html.twig' %}
```

Comment:
Why are the `../` required here. Given the other templates that can simply extend the base template by its name?